ansible/templates/etc/systemd/system/stickypedia.service.j2

```diff
@@ -0,0 +1,22 @@
+# {{ ansible_managed }}
+
+[Unit]
+Description=Run stickypedia (gollum)
+After=network.target
+OnFailure=failure-notificator@%n.service
+
+[Service]
+Type=simple
+User=stickypedia
+Group=www-data
+Restart=always
+
+WorkingDirectory=/var/www/stickypedia/stickypedia.{{ canonical_hostname }}
+
+RuntimeDirectory=wiki
```

Comment:
This will create `/run/wiki`, which is probably not needed.

Reply:
fixed